app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -405,6 +409,32 @@ public void selectAllPhotosUpTo(int targetIndex, MediaAdapter adapter) {
         toolbar.setTitle(selectedMedias.size() + "/" + size);
     }
 
+    public void selectAllPhotosUpToFav(int targetIndex)
+    {
```

Comment:
@angmas1 move the opening bracket of the method next to the method declaration as it is in all other methods.